pkg/controller/pagerdutyintegration/pagerdutyintegration_controller.go

```diff
@@ -162,9 +168,17 @@ type ReconcilePagerDutyIntegration struct {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcilePagerDutyIntegration) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	start := time.Now()
+
 	r.reqLogger = log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	r.reqLogger.Info("Reconciling PagerDutyIntegration")
 
+	defer func() {
+		dur := time.Since(start)
+		localmetrics.SetReconcileDuration("pagerdutyintegration", dur.Seconds())
```

Comment:
```suggestion
		localmetrics.SetReconcileDuration(controllerName, dur.Seconds())
```

Use the constant here?